src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -149,7 +168,7 @@ HttpStatusCode mongoSubscribeContext
                                              tenant,
                                              xauthToken,
                                              servicePathV,
-                                             requestP->expression.q);
+                                             &restriction);
```

Comment:
Yeah, it might look like that. But it is not.
The problem is not here, but inside processConditionVector where it is sent to another function.